keyboards/redragon/k667/info.json

```diff
@@ -0,0 +1,237 @@
+            {"matrix": [2, 14], "x": 217, "y": 24, "flags": 4},
+            {"matrix": [3, 0], "x": 12, "y": 34, "flags": 4},
+            {"matrix": [3, 1], "x": 30, "y": 34, "flags": 4},
+            {"matrix": [3, 2], "x": 44, "y": 34, "flags": 4},
+            {"matrix": [3, 3], "x": 58, "y": 34, "flags": 4},
+            {"matrix": [3, 4], "x": 71, "y": 34, "flags": 4},
+            {"matrix": [3, 5], "x": 85, "y": 34, "flags": 4},
+            {"matrix": [3, 6], "x": 99, "y": 34, "flags": 4},
+            {"matrix": [3, 7], "x": 112, "y": 34, "flags": 4},
+            {"matrix": [3, 8], "x": 126, "y": 34, "flags": 4},
+            {"matrix": [3, 9], "x": 140, "y": 34, "flags": 4},
+            {"matrix": [3, 10], "x": 153, "y": 34, "flags": 4},
+            {"matrix": [3, 11], "x": 167, "y": 34, "flags": 4},
+            {"x": 180, "y": 34, "flags": 4},
+            {"matrix": [3, 13], "x": 189, "y": 34, "flags": 4},
+            {"matrix": [3, 14], "x": 217, "y": 34, "flags": 4},
+            {"matrix": [4, 0], "x": 8, "y": 44, "flags": 4},
+            {"x": 24, "y": 44, "flags": 4},
+            {"matrix": [4, 2], "x": 38, "y": 44, "flags": 4},
+            {"matrix": [4, 3], "x": 51, "y": 44, "flags": 4},
+            {"matrix": [4, 4], "x": 65, "y": 44, "flags": 4},
+            {"matrix": [4, 5], "x": 78, "y": 44, "flags": 4},
+            {"matrix": [4, 6], "x": 92, "y": 44, "flags": 4},
+            {"matrix": [4, 7], "x": 106, "y": 44, "flags": 4},
+            {"matrix": [4, 8], "x": 119, "y": 44, "flags": 4},
+            {"matrix": [4, 9], "x": 133, "y": 44, "flags": 4},
+            {"matrix": [4, 10], "x": 147, "y": 44, "flags": 4},
+            {"matrix": [4, 11], "x": 160, "y": 44, "flags": 4},
+            {"matrix": [4, 12], "x": 179, "y": 44, "flags": 4},
+            {"matrix": [4, 13], "x": 201, "y": 46, "flags": 4},
+            {"matrix": [4, 14], "x": 217, "y": 44, "flags": 4},
+            {"matrix": [5, 0], "x": 8, "y": 54, "flags": 4},
+            {"matrix": [5, 1], "x": 25, "y": 54, "flags": 4},
+            {"matrix": [5, 2], "x": 43, "y": 54, "flags": 4},
+            {"x": 67, "y": 57, "flags": 4},
+            {"x": 80, "y": 57, "flags": 4},
+            {"x": 94, "y": 54, "flags": 4},
+            {"matrix": [5, 6], "x": 107, "y": 57, "flags": 4},
+            {"x": 120, "y": 57, "flags": 4},
+            {"matrix": [5, 9], "x": 143, "y": 54, "flags": 4},
+            {"matrix": [5, 10], "x": 157, "y": 54, "flags": 4},
+            {"matrix": [5, 11], "x": 170, "y": 54, "flags": 4},
+            {"matrix": [5, 12], "x": 187, "y": 56, "flags": 4},
+            {"matrix": [5, 13], "x": 201, "y": 56, "flags": 4},
+            {"matrix": [5, 14], "x": 214, "y": 56, "flags": 4},
+            {"x": 207, "y": 23, "flags": 8},
+            {"x": 207, "y": 27, "flags": 8}
+        ]
+    },
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0021",
+        "vid": "0x369B"
```

Comment:
This vendor ID appears to belong to iton technology corp.  
https://www.the-sz.com/products/usbid/index.php?v=0x369B&p=&n=

Please use a different vendor ID.   In general, about 0x4000 is unassigned, and the further past it, the better, as the vendor ID's appear to be assigned sequentially.  That or just use `0xFEED`

Reply:
Sure. We are iTon technology corp. We have just obtained authorization to use this vid （0x369B）from the USB-IF Administration. Thanx！